### PR TITLE
Fix expressjwt after upgrade

### PR DESCRIPTION
**Pull request by wgwz**
## Description

Related: https://github.com/regen-network/registry-server/pull/201

The recently applied express-jwt upgrade changed the way their primary import statement works, this PR adjusts our code to that change.

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] targeted `dev` branch
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] submit a PR against `master` branch after this one (and other PRs depending on this one, e.g. on regen-network/regen-web, if applicable) get merged

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by victordelval** [comment]
@wgwz for the case of `express-jwt`, maybe it would be a good idea to use the corresponding type provided by DefinitelyTyped, to type the output of the function instead of "any"...

**Comment by wgwz** [comment]
@victordelval it looks like the definitely typed repo is not up-to-date with the latest express-jwt library, so i don't think this will work unfortunately. thanks for the suggestion though, i'm open to other alternatives if there are any

**Review by haveanicedavid — APPROVED** [review]

**Comment by haveanicedavid** [comment]
@victordelval @wgwz express-jwt is written in TS, so has first class type export. I checked out the branch locally though, and properly typing the return of this function isn't so straightforward... dont' have much time to dig further, but could maybe add a TODO in there if we wanted?

**Review by haveanicedavid — APPROVED** [review]

File: server/middleware/jwt.ts [hunk 1]
```diff
@@ -1,10 +1,8 @@
 import * as jwks from 'jwks-rsa';
-import jwt from 'express-jwt';
+import { expressjwt } from 'express-jwt';
 
-export default function getJwt(
-  credentialsRequired: boolean,
-): jwt.RequestHandler {
-  return jwt({
+export default function getJwt(credentialsRequired: boolean): any {
```
Comment (haveanicedavid) [inline server/middleware/jwt.ts:4]:
```suggestion
// we want to use expressjwt's type here
// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
export default function getJwt(credentialsRequired: boolean) {
```

**Review by victordelval — APPROVED** [review]

File: server/middleware/jwt.ts (hunk 1 above)
Comment (wgwz) [reply server/middleware/jwt.ts]: via 759e7ba96e5660cd4520a638e86c1dd670e18f3c

**Pull request merged**